test/integration/scheduler_perf/scheduler_perf.go

```diff
@@ -460,10 +462,16 @@ type createPodsOp struct {
 	// Defaults to false if not specified.
 	// Optional
 	SkipWaitToCompletion bool
+	// Whether or not to wait for all pods in this op to have a SchedulingGated status.
+	// Optional
+	WaitForGated bool
```

Comment:
We don't need this.

k8s adds the condition at Pod creation, so there is nothing to wait for.

https://github.com/kubernetes/kubernetes/blob/efef32652af0af08a0b9c9bc547a4dce4a95f9f5/pkg/registry/core/pod/strategy.go#L94

Reply:
Deleted. I wanted to be sure scheduler processed these pods. Added a few seconds sleep to make sure the stability of the tests is preserved even if there are some delays